Userland/Libraries/LibWeb/ARIA/AriaData.cpp

```diff
@@ -471,10 +485,13 @@ Optional<AriaOrientation> AriaData::parse_aria_orientation(StringView value)
     return {};
 }
 
-Vector<AriaRelevant> AriaData::parse_aria_relevant(StringView value)
+Vector<AriaRelevant> AriaData::parse_aria_relevant(Optional<String> const& value)
 {
+    if (value.has_value())
+        return {};
```

Comment:
I think this should be the other way around?
```suggestion
    if (!value.has_value())
        return {};
```